tests/analyzers_manager/test_file_scripts.py

```diff
@@ -29,6 +29,7 @@ def get_params(cls):
                 "UnpacMe_EXE_Unpacker",
                 "PEframe_Scan",
                 "Capa_Info",
+                "CapeSandbox"
```

Comment:
comma is missing 😅 . Without it, the 2 strings are merged, causing havoc :P

Reply:
Noted and fixed.